orchestrator/agent_orchestrator.py

```
@@ -0,0 +1,22 @@
+import requests
+
+class AgentOrchestrator:
+    def __init__(self, claude_url, api_key):
+        self.claude_url = claude_url
+        self.api_key = api_key
+
+    def deploy_agent(self, agent_config):
+        headers = {'Authorization': f'Bearer {self.api_key}'}
+        response = requests.post(f'{self.claude_url}/agents', json=agent_config, headers=headers)
```

Ruff (0.15.18)

[error] Probable use of `requests` call without timeout

(S113)